routing/routes.go

```diff
@@ -242,6 +242,20 @@ func NewRoutes() (r *Routes) {
 			handlers.ServiceHealth,
 			true,
 		},
+		{
```

Comment:
IMO, It would be good to keep API documentation updated :book:
We have a recent PR for that updates this area: kiali/kiali#387

Reply:
Thanks for the hint, yes I'll do it